server/src/main/java/org/elasticsearch/common/time/RLocale.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.common.time;
+
+public class RLocale {
+    public static final java.util.Locale INS = new java.util.Locale.Builder()
+        .setLocale(java.util.Locale.ROOT)
+        .setUnicodeLocaleKeyword("fw", "mon").build();
```

Comment:
Please add a comment explaining why this is set to monday, and linking to the appropriate java discussion/code/iso definition.

Reply:
good idea